src/Form/Form.php

```diff
@@ -224,6 +310,40 @@ public function isValid(): bool
         return empty($this->errors()) === true;
     }
 
+    /**
+     * Updates the field props of fields that must no be
+     * translated to be disabled when in multilang setup
+     * and not in the default language
+     *
+     * @param array $fields
+     * @param string|null $language
+     * @return array
+     */
+    protected static function prepareFieldsForLanguage(array $fields, ?string $language = null): array
+    {
+        $kirby = App::instance();
```

Comment:
This should use the `$lazy` argument to ensure that the class can still be used without the CMS:

```suggestion
        $kirby = App::instance(null, true);
```